### ISSUE TITLE
Wrong annotation in formatDateTime - lib/internal/Magento/Framework/Stdlib/DateTime/TimezoneInterface.php

### ISSUE THREAD

**Issue by sanjay-wagento** [magento/magento2#15601]
### Preconditions

1. Magento 2 code.

### Steps to reproduce

1. Observe \Magento\Framework\Stdlib\DateTime\TimezoneInterface::formatDateTime

### Expected result

1. The annotation for **locale** and **timezone** should be **@param string|null $locale** and **@param string|null $timezone**

### Actual result

1. The annotation for **locale** and **timezone** is **@param null $locale** and **@param null $timezone**
```
    /**
     * @param string|\DateTimeInterface $date
     * @param int $dateType
     * @param int $timeType
     * @param null $locale
     * @param null $timezone
     * @param string|null $pattern
     * @return string
     */
    public function formatDateTime(
        $date,
        $dateType = \IntlDateFormatter::SHORT,
        $timeType = \IntlDateFormatter::SHORT,
        $locale = null,
        $timezone = null,
        $pattern = null
    );
```

**Comment by magento-engcom-team** [comment]
Hi @sanjay-wagento. Thank you for your report. 
The issue has been fixed in magento/magento2#15667 by @vgelani in 2.3-develop branch
Related commit(s):
 - [2feaf3049e3b2397ba54aafcaa9ec5ff8cba0d83](https://github.com/magento/magento2/commit/2feaf3049e3b2397ba54aafcaa9ec5ff8cba0d83)

The fix will be available with the upcoming 2.3.0 release.

**Comment by magento-engcom-team** [comment]
Hi @sanjay-wagento. Thank you for your report. 
The issue has been fixed in magento/magento2#15670 by @vgelani in 2.3-develop branch
Related commit(s):
 - [11923bc4439e29bd707226bef8377f2ea458a29a](https://github.com/magento/magento2/commit/11923bc4439e29bd707226bef8377f2ea458a29a)

The fix will be available with the upcoming 2.3.0 release.

**Comment by sidolov** [comment]
Hi @sanjay-wagento. Thank you for your report. 
The issue has been fixed in magento/magento2#15669 by @vgelani in 2.1-develop branch
Related commit(s):
 - [5575523512807ede0d93bb86bde30b792a5cc882](https://github.com/magento/magento2/commit/5575523512807ede0d93bb86bde30b792a5cc882)

The fix will be available with the upcoming 2.1.15 release.

**Comment by sidolov** [comment]
Hi @sanjay-wagento. Thank you for your report. 
The issue has been fixed in magento/magento2#15668 by @vgelani in 2.1-develop branch
Related commit(s):
 - [c633d16c15d1676d4604421d096917f863832601](https://github.com/magento/magento2/commit/c633d16c15d1676d4604421d096917f863832601)

The fix will be available with the upcoming 2.1.15 release.

**Comment by ghost** [comment]
@sanjay-wagento, thank you for your report.
We've acknowledged the issue and added to our backlog.

**Comment by ghost** [comment]
Hi @sanjay-wagento , thank you for you report, this issue has already fixed in 2.3-develop branch, and available on 2.3.0 release.
